packages/frontend/component/src/ui/scrollbar/index.css.ts

```diff
@@ -30,6 +30,9 @@ export const scrollableViewport = style({
 globalStyle(`${scrollableViewport} > div`, {
   display: 'contents !important',
```

Comment:
I forget why I added this. What will happen if this one is removed?

Reply:
Yeah I know. I mean I forget what will happen if using `display: table`.  At least it looks like when it is used, edgeless canvas will display (with height: 0)

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/T2klNLEk0wxLh4NRDzhk/539f5ffe-8a18-4344-95c5-373214f58a99.png)